optuna/study/study.py

```diff
@@ -913,9 +895,6 @@ def objective(trial):
         Args:
             trials: Trials to add.
 
-        Raises:
-            :exc:`ValueError`:
-                If ``trials`` include invalid trial.
```

Comment:
Ditto.

Reply:
After discussion for `add_trial`, I'll update here on demand.